### PR TITLE
[frontend] add support for antd modal component

### PR DESCRIPTION
**Pull request by bjornalm**
## What changes were proposed in this pull request?

- Modifies the css to correctly show the Antd modal
- Updates the react example component with a modal opened by clicking a button

## How was this patch tested?
Manually tested

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

File: desktop/core/src/desktop/static/desktop/css/hue.css [hunk 1]
```diff
@@ -14,4 +14,4 @@ distributed under the License is distributed on an "AS IS" BASIS,
...
...color:#d6d8db}tr.even.gradeU td.sorting_2{background-color:#eaebec}tr.even.gradeU td.sorting_3{background-color:#eaebec}.ex_highlight #example tbody tr.even:hover,#example tbody tr.even td.highlighted{background-color:#ffd664}.ex_highlight #example tbody tr.odd:hover,#example tbody tr.odd td.highlighted{background-color:#ffd664}.ex_highlight_row #example tr.even:hover{background-color:#ffd664}.ex_highlight_row #example tr.even:hover td.sorting_1{background-color:#ffd664}.ex_highlight_row #example tr.even:hover td.sorting_2{background-color:#ffd664}.ex_highlight_row #example tr.even:hover td.sorting_3{background-color:#ffd664}.ex_highlight_row #example tr.odd:hover{background-color:#ffd664}.ex_highlight_row #example tr.odd:hover td.sorting_1{background-color:#fdc926}.ex_highlight_row #example tr.odd:hover td.sorting_2{background-color:#ffd664}.ex_highlight_row #example tr.odd:hover td.sorting_3{background-color:#ffd664}table.KeyTable td{border:3px solid transparent}table.KeyTable td.focus{border:3px solid #607D8B}table.display tr.gradeA{background-color:#EDF5E2}table.display tr.gradeC{background-color:#d5c9ed}table.display tr.gradeX{background-color:#fff2f2}table.display tr.gradeU{b
```
Comment (JohanAhlen) [inline desktop/core/src/desktop/static/desktop/css/hue.css:17]: Was there no change to hue.less?

Comment (JohanAhlen) [reply desktop/core/src/desktop/static/desktop/css/hue.css:17]: i.e. import of root-wrapped-antd.less somewhere?

Comment (bjornalm) [reply desktop/core/src/desktop/static/desktop/css/hue.css:17]: Nope, root-wrapped-antd.less is already imported into hue.less from an earlier PR https://github.com/cloudera/hue/pull/2996/files#diff-7ddb051492779805fec518b02b60d805888a89b4d4e86114528815dda8dbe21b

**Review by JohanAhlen — APPROVED** [review]
LGTM!

**Pull request merged**